browser/ui/views/frame/vertical_tab_strip_region_view.cc

```diff
@@ -1029,10 +1030,12 @@ void VerticalTabStripRegionView::ResetExpandedWidth() {
   PreferredSizeChanged();
 }
 
-void VerticalTabStripRegionView::UpdateTabSearchButtonVisibility() {
+void VerticalTabStripRegionView::UpdateOriginalTabSearchButtonVisibility() {
   const bool is_vertical_tabs = tabs::utils::ShouldShowVerticalTabs(browser_);
+  const bool use_search_button =
```

Comment:
Just double checking that we don't need to listen for changes to the pref?

Reply:
Yeah, for now we don't hide or show the tab search button on top of vertical tab strip. We haven't decided. So we don't need to observe it for now.